calico.go

```diff
@@ -339,16 +367,6 @@ func cmdDel(args *skel.CmdArgs) error {
 		logger.Error(ipamErr)
 	}
 
-	calicoClient, err := CreateClient(conf)
-	if err != nil {
-		return err
-	}
-
-	ep := api.WorkloadEndpointMetadata{
-		Name:         args.IfName,
-		Node:         nodename,
-		Orchestrator: orchestrator,
-		Workload:     workload}
 	if err = calicoClient.WorkloadEndpoints().Delete(ep); err != nil {
```

Comment:
You probably want to explicitly handle an error type of `errors.ErrorResourceUpdateConflict` - I'm not sure if you need to retry the operation (I can't think that you'd need to), but you may want to log out that we're no-oping due to a concurrent update.